src/homework/client/src/reducers/items-view-reducer.js

```diff
@@ -0,0 +1,12 @@
+import actionTypes from "../actions/constants";
+
+const initialState = {
+};
+
+const itemsViewReducer = (state = initialState, action) => {
+  switch (action.type) {
```

Comment:
the reducer is empty.
why didnt you add the actions you create?
HIDE_LOADER, SHOW_LOADER